std_msgs/Header header

ars_40X/Object[] objects